bcipy/helpers/visualization.py

```diff
@@ -197,6 +397,8 @@ def visualize_session_data(session_path: str, parameters: dict, show=True) -> Fi
     filter_order = parameters.get("filter_order")
     static_offset = parameters.get("static_trigger_offset")
 
+    # TODO check for existence of these files and throw error if not found
```

Comment:
Is this still relevant?

Reply:
The comments were outdated, I deleted them